frontend/src/v5/ui/components/shared/jobPopover/jobPopover.styles.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import styled from 'styled-components';
+import { JobAvatar as JobAvatarBase } from '@controls/jobAvatar/jobAvatar.component';
+
+export const JobAvatar = styled(JobAvatarBase)`
+	.MuiAvatar-root {
+		background-color: ${({ theme }) => theme.palette.base.main};
+		color: ${({ theme }) => theme.palette.primary.contrast};
```

Comment:
I think in both places jobPopover.component.tsx and asignees are using the same style. Instead of doing this I think it would be better if we change it in jobAvatar.component